core/controllers/admin.py

```diff
@@ -585,13 +685,43 @@ class AdminRoleHandler(base.BaseHandler):
     """Handler for roles tab of admin page. Used to view and update roles."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'filter_criterion': {
+                'type': 'string',
+                'choices': ['role', 'username']
```

Comment:
Please use the feconf constants(USER_FILTER_CRITERION_ROLE, USER_FILTER_CRITERION_USERNAME) instead of hardcoding it.

Reply:
Done